### PR TITLE
Do not tildify URIs

### PR DESCRIPTION
**Pull request by lgeiger**
See https://github.com/atom/status-bar/pull/171

### CODE REVIEW

File: spec/fs-plus-spec.coffee [hunk 1]
```diff
@@ -516,6 +516,9 @@ describe "fs", ->
         expect(fs.tildify(fixture)).toBe fixture
         expect(fs.tildify('foo')).toBe 'foo'
 
+    it "should not tildify URIs", ->
+      uri = 'remote://server:123/folder/remote_file.txt'
+      expect(fs.tildify(uri)).toBe uri
```
Comment (lgeiger) [inline spec/fs-plus-spec.coffee]: I would like to spy on the tildify module to ensure it isn't called but I don't know if there is a easy way to spy on a required module in jasmine.

**Comment by damieng** [comment]
Sorry for all the back and forth on this - I think it's worth not taking the tildify dependency after all.  I've commited a tildify function that does exactly what we want including not modifying the path at all if it won't tildify it.  Will npm publish this soon as 2.9.3 and then we can get the status-view one merged.

**Comment by lgeiger** [comment]
@damieng It looks like 2.9.3 is only published on github and not on npm yet.

**Comment by damieng** [comment]
Yeah, I needed to get some permissions :)  Now published.

**Comment by lgeiger** [comment]
Thanks! I'll update my PR

**Pull request closed without merging**